geode-core/src/integrationTest/java/org/apache/geode/distributed/LocatorLauncherIntegrationTestCase.java

```diff
@@ -157,7 +156,7 @@ private String getClusterConfigDirectoryPath() {
 
   private boolean isLauncherOnline() {
     LocatorState locatorState = launcher.status();
-    assertNotNull(locatorState);
+    assertThat(locatorState).isNotNull();
```

Comment:
Would we also want to maybe assert WHAT state the locator is in, or do we not care?